ethereum/contracts/Messages.sol

```diff
@@ -26,9 +26,45 @@ contract Messages is Getters {
     *  - it aims to verify the signatures provided against the guardianSet
     */
     function verifyVM(Structs.VM memory vm) public view returns (bool valid, string memory reason) {
+        (valid, reason) = verifyVMInternal(vm, true);    
+    }
+
+    /**
+    * @dev `verifyVMInternal` serves to validate an arbitrary vm against a valid Guardian set
+    *  - it is similar to verifyVM but drops the verification of the hash as it is called on the result of parseVM whose vm.hash field can be trusted.
```

Comment:
this is outdated now because of the argument

Reply:
updated the comments now